gnomad/resources/grch38/gnomad.py

```diff
@@ -19,17 +19,20 @@
 DATA_TYPES = ["genomes"]
 
 GENOME_POPS = ["AFR", "AMI", "AMR", "ASJ", "EAS", "FIN", "NFE", "SAS", "OTH"]
-SUBSETS = [
-    "non_v2",
-    "non_topmed",
-    "non_cancer",
-    "controls_and_biobanks",
-    "non_neuro",
-    "tgp",
-    "hgdp",
-]
+SUBSETS = {
+    "v3": [
+        "non_v2",
+        "non_topmed",
+        "non_cancer",
+        "controls_and_biobanks",
+        "non_neuro",
+        "tgp",
+        "hgdp",
+    ],
+    "v4": ["ukb", "non-ukb", "non-topmed"],
```

Comment:
Just confirming there is an intentional switch to `-` from `_` I don't remember all thee fine details associated with that.

Reply:
Ah no good catch. The 4.3 vcf spec does not allow hyphens so underscores are better here to be consistent with the vcf. I took this straight from the v4 release HT schema but I'll update it there as well.

Reply:
https://atgu.slack.com/archives/CNL7NA6H2/p1604673884093000?thread_ts=1604652743.092300&cid=CNL7NA6H2